src/lang/en.json

```diff
@@ -645,6 +645,7 @@
     "pushyToken": "Device token",
     "apprise": "Apprise (Support 50+ Notification services)",
     "GoogleChat": "Google Chat (Google Workspace only)",
+    "Template plain text instead of using cards": "Template plain text instead of using cards",
```

Comment:
```suggestion
    "Template plain text instead of using cards": "Template plain text instead of using cards",
    "issueWithGoogleChatOnAndroidHelptext": "This also allows to get around bugs upstream like {issuetackerURL}",
```